Sieve.md

```diff
@@ -1,5 +1,5 @@
 ## See also
-* [Sieve of Eratosthenes]()
+* [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes)
```

Comment:
An empty link, such as `[Sieve of Eratosthenes]()`, links to the internal page called "Sieve of Eratosthenes". I think this link should remain internal, and instead the wiki link should go under "External links" on the internal "Sieve of Eratosthenes" page.